tests/lib/rules/no-duplicate-dependent-keys.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/no-duplicate-dependent-keys');
+const RuleTester = require('eslint').RuleTester;
+
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester();
+const parserOptions = { ecmaVersion: 6, sourceType: 'module' };
+ruleTester.run('no-duplicate-dependent-keys', rule, {
+  valid: [
+    {
+      code: `
+      {
+        foo: computed('model.foo', 'model.bar', 'model.baz', function() {})
```

Comment:
We should test more cases, those with `@each` and `[]` too. People tend to create really crazy dependent keys and we need to make sure we support all of those weird cases, so I recommend to enlarge the test suite for this rule significantly :) WDYT?